spec/system/candidate_interface/candidate_does_not_know_where_they_want_to_apply_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.feature 'Selecting a course' do
+  include CandidateHelper
+
+  scenario 'Candidate selects a course choice' do
```

Comment:
```suggestion
  scenario 'Candidate does not know what course to apply for' do
```

As the spec file name suggests and feels more representative of what we are testing?

Reply:
Good shout.